src/Components/FormPage/FormPage.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react';
+import './FormPage.css';
+import arrow from '../../images/arrow.png';
+import { postNewForm } from '../../apiCalls';
+import Question1 from './Questions/Question1';
+import Question2 from './Questions/Question2';
+import Question3 from './Questions/Question3';
+import Question4 from './Questions/Question4';
+import {QuestionComponents, TechStacks, ErrorConditions, TimeFrame, ErrorMsg, FormData} from './FormPageTypes';
+
+const FormPage = () => {
+  const [currentQuestion, setCurrentQuestion] = useState<number>(1);
+  const [stack, setStack] = useState<string>("");
+  const [searchTerm, setSearchTerm] = useState<string>("");
+  const [technologies, setTechnologies] = useState<string[]>([]);
+  const [timeframe, setTimeframe] = useState<TimeFrame>({amount: 0, type: "days"});
+  const [numPeople, setNumPeople] = useState<number>(0);
+  const [error, setError] = useState<ErrorMsg>({error: false, message: ""})
+
+  const questions: Record<number, string> = {
+    1: "Choose Your Application Type",
+    2: "Which Technologies Would You Like to Use?",
+    3: "How Long Do You Have to Create This App?",
+    4: "How Many Collaborators Will There Be?"
+  }
+
+  const techStacks: TechStacks = {
+    frontend: ['react', 'typescript', 'javascript', 'vue', 'angular'],
+    backend: ['ruby/rails', 'postgresql', 'node', 'sidekiq', 'devise'],
+    fullstack: ['react', 'typescript', 'javascript', 'vue', 'angular', 'ruby/rails', 'postgresql', 'node', 'sidekiq', 'devise']
+  }
+
+  const questionComponents: QuestionComponents = {
+    1: <Question1 chooseStack={(e) => chooseStack(e)} />,
+    2: <Question2 setTechnologies={setTechnologies} searchTerm={searchTerm} techStacks={techStacks} setError={setError} stack={stack} setSearchTerm={setSearchTerm} technologies={technologies}/>,
+    3: <Question3 setTimeframe={setTimeframe} timeframe={timeframe}/>,
+    4: <Question4 numPeople={numPeople} setNumPeople={setNumPeople}/>
+  }
```

Comment:
In regards to passing the setter functions down I think it is fine. Keeping the state in the parent component is probably also best for the ability to have the answers persist when you go back to previous questions, so I like what you did!